src/server/auth-token.js

```diff
@@ -61,21 +61,22 @@ class AuthToken {
   static authenticate(req, res, next) {
     const bearerToken = req.headers.authorization;
     if (!bearerToken) {
-      return next();
+      next();
+      return;
```

Comment:
This method does not have to return promise.
This method's goal is just call next() function after done its jobs.

But we need to keep consistency. I mean if it returns promise in some case and does not return promise (normal function) in some case, It is not consistent.